classes/helper/HelperOptions.php

```diff
@@ -173,13 +173,16 @@ public function generateOptions($option_list)
 
                 // Fill values for all languages for all lang fields
                 if (substr($field['type'], -4) == 'Lang') {
+                    $field['value'] = array();
                     foreach ($languages as $language) {
                         if ($field['type'] == 'textLang') {
                             $value = Tools::getValue($key.'_'.$language['id_lang'], Configuration::get($key, $language['id_lang']));
                         } elseif ($field['type'] == 'textareaLang') {
                             $value = Configuration::get($key, $language['id_lang']);
                         } elseif ($field['type'] == 'selectLang') {
                             $value = Configuration::get($key, $language['id_lang']);
+                        } else {
+                            continue;
```

Comment:
You are potentially modifying the behavior of PrestaShop with this `else`, because some new input (created by module ?) would not appear anymore.

Can you describe the issue you try to fix here ?